.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+Thank you for taking the time to file an issue, we highly appreciate your contribution.
+
+Before you go ahead, please make sure that the issue has not been reported already.
+For support questions, please head over to our [Discord](http://near.chat/) or ask on [StackOverflow](https://stackoverflow.com/search?q=nearprotocol).
+-->
+
+**Describe the bug**
+Please provide a short description of the bug.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```

Comment:
How is smartphone relevant?

Reply:
Good question -- e.g. website, docs and products may have bugs on smartphone that are not visible on the browser in terms of UI/UX

Reply:
But that is not relevant to nearcore.

Reply:
So I will create separate templates for the other repos